73_addons/5_using_v4l2_rtsp_server_to_stream.md

```diff
@@ -0,0 +1,91 @@
+## Using V4L2 and RTSP server
+The following shows how to use V4L2 capture with a RTSP server to stream from the Raspberry Pi camera to Kerberos.io running somewhere else (In my case kerberos.io is running in a docker container on a central server)
```

Comment:
I would not say "My case", but make it more general.

"from the Raspberry Pi camera to Kerberos.io (for example on another Raspberry Pi or Docker container)